core/connection.go

```diff
@@ -188,6 +209,11 @@ func createConnectionStep(src, dst *ProvableChain) (*RelayMsgs, error) {
 		out.Last = true
 
 	default:
+		connectionErrorw(
+			zapLogger,
+			"not implemented",
+			fmt.Errorf("not implemented error: %v %v", srcConn.Connection.State, dstConn.Connection.State),
+		)
```

Comment:
This switch case is basically unreachable, so it not necessary to put a new logging here.

Reply:
Remove not necessary code
https://github.com/hyperledger-labs/yui-relayer/pull/92/commits/2cd89343a2eb439b4ab72ae721bf841457c68895